.github/workflows/cross-version-tests.yml

```diff
@@ -103,11 +103,19 @@ jobs:
         with:
           repository: ${{ github.event.inputs.repository }}
           ref: ${{ github.event.inputs.ref }}
+      - name: Get Java version
+        id: get-java-version
+        run: |
+          if [ "${{ matrix.package }}" = "mleap" ]
```

Comment:
mleap still uses spark 2.4.

Reply:
The latest mleap version already support spark 3.x ?

Reply:
@WeichenXu123 

https://github.com/combust/mleap#requirements says:

> MLeap is built against Scala 2.11 and Java 8. Because we depend heavily on Typesafe config for MLeap, we only support Java 8 at the moment.

Reply:
Until MLeap supports >3.1.x the solution in this PR will be needed for the netty serialization issue, right?

Reply:
Yes!